### DISCUSSION TITLE
配置文档缺失

### DISCUSSION THREAD

**Discussion by xiaohp** [umijs/umi#9001] [General]
https://umijs.org/docs/api/config

有些文档中的选项，在配置里找不到，比如 https://umijs.org/docs/tutorials/blog 这里用到的 
- npmClient
- apiRoute
- tailwindcss

另外教程文档的 demo 加载不出来
https://umi-blog-example.vercel.app/
![image](https://user-images.githubusercontent.com/23056320/185082009-2e8950d3-b115-44fa-97d1-4599f796923a.png)

**Comment by fz6m** [comment]
`apiRoute` / `tailwindcss` 是插件提供的，不在内置文档内。

`npmClient` 的作用是安装依赖时首选的包管理器，欢迎 PR 补充。

**Comment by sanqianmenglili** [comment]
apiRoute  是内置的插件吗    为什么官方的文档中 查不到这个插件，另外官方的文档中关于 apiRoute 的相关内容除了博客这一栏，也没有别的介绍了
